tasks/taxon_id/task_nextclade.wdl

```diff
@@ -60,6 +60,72 @@ task nextclade {
   }
 }
 
+task nextclade_v3 {
+  meta {
+    description: "Nextclade classification of one sample. Leaving optional inputs unspecified will use SARS-CoV-2 defaults."
+  }
+  input {
+    File genome_fasta
+    File? auspice_reference_tree_json
+    File? gene_annotations_gff
+    File? pcr_primers_csv
+    File? nextclade_pathogen_json
+    String docker = "us-docker.pkg.dev/general-theiagen/nextstrain/nextclade:3.3.1" 
+    String dataset_name
+    String verbosity = "warn" # other options are: "off" "error" "info" "debug" and "trace"
+    String dataset_tag
+    Int disk_size = 50
+    Int memory = 4
+    Int cpu = 2
+  }
+  String basename = basename(genome_fasta, ".fasta")
+  command <<<
+    # track version & print to log
+    nextclade --version | tee NEXTCLADE_VERSION
+
+    # --reference no longer used in v3. consolidated into --name and --tag
+    nextclade dataset get \
+      --name="~{dataset_name}" \
+      --tag="~{dataset_tag}" \
+      -o nextclade_dataset_dir \
+      --verbosity ~{verbosity}
+
+    # exit script/task upon error
+    set -e
+
+    # not necessary to include `--jobs <jobs>` in v3. Nextclade will use all available CPU threads by default. It's fast so I don't think we will need to change unless we see errors
+    nextclade run \
+      --input-dataset nextclade_dataset_dir/ \
+      ~{"--input-tree " + auspice_reference_tree_json} \
+      ~{"--input-pathogen-json " + nextclade_pathogen_json} \
+      ~{"--input-annotation " + gene_annotations_gff} \
+      ~{"--input-pcr-primers " + pcr_primers_csv} \
```

Comment:
Looks as if `--input-pcr-primers` was also removed as an [input flag](https://docs.nextstrain.org/projects/nextclade/en/stable/user/migration-v3.html#url-parameters-in-nextclade-web-have-changed)

Reply:
the `--input-root-seq` flag (and associated `root_sequence`task input) were removed from the task, but looks as if these were just renamed in nextclade v3 ^^ info in same docs linked above.

Reply:
good catch, thank you. I will remove the `--input-pcr-primers` and add the `--input-root-seq` to `--input-ref` as specified in their docs

Reply:
resolved in [2b7470a](https://github.com/theiagen/public_health_bioinformatics/pull/375/commits/2b7470ab57f459788f33c606441446e8043bae50)